projects/grafana/setup-grafana.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+DS_NAME=$1
+DASH_FILE="./openshift-cluster-monitoring.json"
+
+oc create namespace grafana
+#TODO: we probably wanna use the following commands to deploy the env
```

Comment:
@stevekuznetsov @kargakis PTAL
also, we may want to join the grafana.yaml and the grafana-ocp.yaml (which is the one that creates the pod) into single yaml

Reply:
Yes, remove the RC and add the DC in `grafana-ocp.yaml`.